bot.py

```diff
@@ -597,7 +613,9 @@ async def shulkify(ctx, count: int):
 @bot.command(name="toggle")
 # Checks that user is Harvite
 @commands.has_role(999078830973136977)
-async def toggle(ctx, feature):
+async def toggle(
+    ctx, feature: any = commands.parameter(description="The feature to toggle")
```

Comment:
This seems rather obvious. Descriptions are not always necessary.

Reply:
same goes for many of the other descriptions you added

Reply:
To prevent automatic placeholders "No description given", which looks bad.